modules/ept-frontend/client/services/websocket.js

```diff
@@ -124,7 +124,6 @@ function(Alert, Auth, NotificationSvc, Session, $window, $rootScope) {
   });
 
   return {
-    init: function() {},
```

Comment:
is this function being called anywhere?  just checking.

Reply:
No

Reply:
kk thx

Reply:
![image](https://user-images.githubusercontent.com/3689811/31524932-7966a084-af58-11e7-89f3-9eb38880b69b.png)